src/core/stack.c

```diff
@@ -202,6 +237,7 @@ meta_stack_freeze (MetaStack *stack)
   stack->freeze_count += 1;
 }
 
+
```

Comment:
Again, why adding a blank line ?

Reply:
Can this be done without an extra commit please?
`git rebase -i <last-commit-id-before-PR>
This will open vi editor in terminal, select commit and choose `edit`

Reply:
Hope its ok now

Reply:
no, i am given up.....

Reply:
Same here :P